public/static/locales/en/header.json

```diff
@@ -9,5 +9,6 @@
 
   "listings": "Listings",
   "home": "Home",
-  "travel": "Travel Directory"
+  "travel": "Travel Directory",
+  "mobile": "Back to Homepage"
```

Comment:
Minor point: 'Homepage' is not capitalised in the TP requirement. (The Welsh equivalent is correct).

Reply:
Corrected. Thanks.